packages/@tailwindcss-upgrade/src/codemods/migrate-tailwind-directives.test.ts

```diff
@@ -400,3 +400,37 @@ it('should drop `@tailwind variants;`', async () => {
     `),
   ).toEqual('')
 })
+
+it('should replace `@variants` with its children', async () => {
+  expect(
+    await migrate(css`
+      @variants hover, focus {
+        .foo {
+          color: red;
+        }
+      }
+    `),
+  ).toMatchInlineSnapshot(`
+    "@variants hover, focus {
+      .foo {
+        color: red;
+      }
+    }"
```

Comment:
Either the test name is wrong or the snapshot is wrong

Reply:
Deleted it now; This was moved to a separate file. Should be all good now.